index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Profile</title>
+
+    <link rel="stylesheet" href="./bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
+
+    <script src="http://code.jquery.com/jquery-3.3.1.min.js"integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="crossorigin="anonymous"></script>
+    <script src="bootstrap/js/bootstrap.min.js" charset="utf-8"></script>
+
+  </head>
+  <body>
+    <header>
+      <nav class="navbar navbar-fixed-top">
+         <div class="container">  <!--/containder start -->
+           <div class="navbar-header"> <!--navbarheader start -->
+             <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+               <span class="sr-only">Toggle navigation</span>
+               <span class="icon-bar"></span>
+               <span class="icon-bar"></span>
+               <span class="icon-bar"></span>
+             </button>
+             <a class="navbar-brand" href="#">Profile</a>
+           </div> <!--navbar head end -->
+           <div class="navbar" class="collapse navbar-collapse"> <!--/collapse start -->
+             <ul class="nav navbar-nav">
+               <li class="active"><a href="#">Alloys</a></li>
```

Comment:
This nav bar on the homepage has links that don't actually go to the place they're supposed to.